g/guava-guava/build_info.json

```diff
@@ -0,0 +1,15 @@
+{
+    "maintainer": "santhoshkotla2929",
+    "package_name": "guava-guava",
```

Comment:
Can name package as guava. Please update folder, script and any other place applicable.